src/app/component/NewProducts/NewProducts.container.js

```diff
@@ -138,7 +165,30 @@ export class NewProductsContainer extends PureComponent {
             );
     }
 
-    render = () => <NewProducts { ...this.props } { ...this.state } />;
+    render() {
+        const {
+            siblingsHaveBrands,
+            siblingsHavePriceBadge,
+            siblingsHaveTierPrice,
+            siblingsHaveConfigurableOptions
+        } = this.state;
+
+        const productCardProps = {
+            siblingsHaveBrands,
+            siblingsHavePriceBadge,
+            siblingsHaveTierPrice,
+            siblingsHaveConfigurableOptions
+        };
```

Comment:
Throughout the application such stuff is declared as member functions on classes, e.g.
```js
containerProps = () => {
        const {
            isActive,
            itemRef,
            children
        } = this.props;

        return {
            isActive,
            itemRef,
            children
        };
    };
```

Reply:
Moved these props to containerProps().